docs/src/piccolo/migrations/create.rst

```diff
@@ -230,6 +230,11 @@ especially if the table is large, with many foreign keys.
 
         return manager
 
+.. warning:: For MySQL we need to run migrations outside transaction due to 
+    MySQL limitations. We can achive that by setting the ``MigrationManager`` 
+    argument ``wrap_in_transaction`` to ``False`` so that the migration 
+    is not wrapped in a transaction.
```

Comment:
I think this is fine - but we might be able to automatically do this by detecting that MySQL is the engine type.